symphony/app/fbcnms-projects/magmalte/views/login.pug

```diff
@@ -11,6 +11,7 @@ html
     meta(name="description", content="Magma NMS")
     meta(name="viewport", content="width=device-width, initial-scale=1")
     link(rel="shortcut icon" type="image/png" href="/nms/static/images/favicon.png")
+    link(href='https://fonts.googleapis.com/css2?family=Fira+Code:wght@500&family=Inter:wght@500;600;700&display=swap' rel='stylesheet')
```

Comment:
hmm is this how we should link to fonts? haven't seen us do this anywhere else, let's ask about it

Reply:
I guess my issue is whether we should be importing this font from Google or serving it with our own static resources. Is it standard industry practice to just link to the google fonts? are those URLs guaranteed to never break?